src/main/java/org/openbase/bco/bcozy/controller/powerterminal/chartattributes/VisualizationType.java

```diff
@@ -0,0 +1,10 @@
+package org.openbase.bco.bcozy.controller.powerterminal.chartattributes;
+
+public enum VisualizationType {
+    BAR, PIE, WEBVIEW, LINECHART;
+
+    @Override
+    public String toString() {
+        return super.toString().substring(0, 1) + super.toString().substring(1).toLowerCase();
```

Comment:
see above

Reply:
Reworked in 62de25f and removed unused functions in 1090e0b.